docs/quickstart.md

```diff
@@ -135,46 +173,76 @@ def cross_entropy(out, Y):
   y = y.reshape(list(Y.shape)+[num_classes])
   y = Tensor(y)
   return out.mul(y).mean()
+
 ```
+
 As we can see in this implementation of cross entropy loss, there are certain operations that tinygrad does not support.
 Namely, operations that are load/store like indexing a tensor with another tensor or assigning a value to a tensor at a certain index.
 Load/store ops are not supported in tinygrad because they add complexity when trying to port to different backends and 90% of the models out there don't use/need them.
 
 For our optimizer we will be using the traditional stochastic gradient descent optimizer with a learning rate of 3e-4.
+
 ```py
 from tinygrad.nn.optim import SGD
 
-opt = SGD([net.l1.weight, net.l2.weight], lr=3e-4)
+#opt = SGD([net.l1.weight, net.l2.weight], lr=3e-4)
```

Comment:
Why was this commented out?

Reply:
I did not want to remove it but at the same time it explained what was sent to SGD in the call below so it ended up with a comment

Reply:
I think keeping the SGD here and removing the one below would be better because this one is more explicit.

Reply:
done